src/earthdataVarinfo/handler.py

```diff
@@ -0,0 +1,70 @@
+import os
+import traceback
+import json
+import shutil
+from tempfile import mkdtemp
+
+from varinfo import VarInfoFromNetCDF4
+from varinfo.cmr_search import get_granules, get_granule_link, download_granule
+from varinfo.umm_var import export_all_umm_var_to_json, get_all_umm_var
+
+
+def main(event, context):
```

Comment:
Super nit: This function has no documentation string, and probably should.

Reply:
Will add